File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L1/01/L1.ts
import { readlineSync } from 'readline-sync';

function reverse(number: number): void {
  while (number !== 0) {
    const remainder = number % 10;
    console.log(remainder);
    number = Math.floor(number / 10);
  }
  console.log();
}

function main(): void {
  console.log('Enter an integer: ');
  const number = readlineSync.questionInt();
  reverse(number);
}

main();

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L1/01/L1.kt
package IR

fun main() {
    println("Enter an integer: ")
    val input = readln().toInt()
    reverse(input)
}

fun reverse(number: Int) {
    var num = number
    while (num != 0) {
        val remainder = num % 10
        print(remainder)
        num /= 10
    }
    println()
}

